ansible/inventories/mgmt/group_vars/all/vars.yml

```diff
@@ -1,3 +1,17 @@
 # TrendMicro
 trendmicro_policy_id_mgmt_app: "{{ vault_trendmicro_policy_id_mgmt_app }}"
 trendmicro_policy_id_mgmt_web: "{{ vault_trendmicro_policy_id_mgmt_web }}"
+
+# Elasticsearch/Kibana shared creds
+kibana_elasticsearch_username: "{{ vault_kibana_elasticsearch_username }}"
+kibana_elasticsearch_password: "{{ vault_kibana_elasticsearch_password }}"
+elasticsearch_name: "{{ vault_elasticsearch_name }}"
+elasticsearch_port: 9200
+kibana_port: 5601
+
+# Nginx basic auth
+basic_auth_user: "{{ vault_basic_auth_user }}"
+basic_auth_hash: "{{ vault_basic_auth_hash }}"
```

Comment:
I didn't see these in the vault, maybe they are missing the `vault_` prefix in the vault?

Reply:
Good catch, added a commit to prepend the missing `vault_`